src/command_modules/azure-cli-acr/HISTORY.rst

```diff
@@ -5,6 +5,7 @@ Release History
 
 2.0.23
 ++++++
+* Update table columns and ordering.
```

Comment:
Change version to 2.0.24 and update setup.py.
The build for 2.0.23 has already been created.